heufybot/modules/commands/log.py

```diff
@@ -39,8 +39,11 @@ def execute(self, server, source, command, params, data):
             logDate = date.today() - timedelta(delta)
         else:
             try:
-                formattedDate = params[0].split("-")
-                logDate = date(int(formattedDate[0]), int(formattedDate[1]), int(formattedDate[2]))
+                slashesAndDashes = ["/", "-"]
+                for symbol in slashesAndDashes:
```

Comment:
You can change this to simply `for symbol in ["/", "-"]`. No need to store it in a variable first.